app/views/static_pages/index.html.erb

```diff
@@ -1,2 +1,9 @@
 <h1>Hello Home </h1>
+<% if current_user %>
+  <% if current_user.profile_photo.file.nil? %>
+    <p>No Profile Photo </p>
+  <%else%>
+    <img src=<%= current_user.profile_photo %> height="60" width="60">
```

Comment:
I would break this out into a CSS file for the whole feature page, instead of specifying the height and width here.